server/lib/tuist_web/marketing/controllers/marketing_html/home.html.heex

```diff
@@ -1,15 +1,15 @@
 <!-- Hero section -->
 <div class="marketing__home__section__hero">
   <h1 class="font-xxxl-strong marketing__home__section__hero__title">
-    {gettext("A virtual platform team for mobile devs who ship")}
+    {gettext("Make mobile your competitive advantage")}
   </h1>
   <p class="marketing__home__section__hero__subtitle">
     {gettext(
-      "We are an integrated and open core toolchain that extends Apple's official tools with insights, optimizations, and workflows to help you build better apps faster"
+      "The same iOS tooling that powers billion-user apps, delivered as a service for your team"
```

Comment:
```suggestion
      "The same iOS tooling that powers million-user apps, delivered as a service for your team"
```

I don't think we're powering billion-user apps (of which there are just a couple in the world 😁)